### PR TITLE
Add tests for onboarding context

### PR DESCRIPTION
**Pull request by amandabeiner**
Why:
As a team, we want to be confident that the app's onboarding behavior is
acting as expected.

This commit:
Adds a spec for the OnboardingContext

### CODE REVIEW

File: src/OnboardingContext.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+      expect(storageSpy).toHaveBeenCalled()
+      expect(getByText(/onboarding complete/)).toHaveTextContent("false")
+    })
+  })
+})
+
+const OnboardingStatus: FunctionComponent = () => {
+  const { isOnboardingComplete } = useOnboardingContext()
+  console.log("in spec component", { isOnboardingComplete })
```
Comment (johnschoeman) [inline src/OnboardingContext.spec.tsx]: extra console.log?

**Review by johnschoeman — APPROVED** [review]
I dig the test style. I usually hook up dummy button and exercise them with a fireEvent call, the useEffect is a neat alternative.

File: src/OnboardingContext.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        </OnboardingProvider>,
+      )
+
+      expect(storageSpy).toHaveBeenCalled()
+      expect(getByText(/onboarding complete/)).toHaveTextContent("true")
+    })
+  })
+
+  describe("resetting onboarding", () => {
+    it("sets onboarding incomplete in storage and context", () => {
```
Comment (devinjameson) [inline src/OnboardingContext.spec.tsx]:
```suggestion
    it("sets onboarding to incomplete in storage and context", () => {
```

**Review by devinjameson — APPROVED** [review]
Very interesting, haven't seen any tests like this before but I dig it!

**Pull request merged**